src/components/ModalWithConfirm.js

```diff
@@ -0,0 +1,20 @@
+import Modal from "./Modal";
+
+export default class ModalWithConfirm extends Modal {
```

Comment:
Great job implementing confirmation popup and the functionality related to it! In general, it can support any confirmation.